DEVELOPERS.md

```diff
@@ -52,111 +54,70 @@ To contribute code to Supabase, you must fork the [Supabase Repository](https://
    cd supabase
    ```
 
-### Choose a directory
-
-Choose if you want to work on the [Supabase Website](https://supabase.com), [Supabase Docs](https://supabase.com/docs), or [Supabase Studio](https://app.supabase.com).
-
-1. Go to the [Supabase Website](https://supabase.com) directory
-
-   ```sh
-   cd apps/www
-   ```
-
-   Go to the [Supabase Docs](https://supabase.com/docs) directory
+### Running turborepo
 
-   ```sh
-   cd apps/reference
-   ```
-
-   Go to the [Supabase Studio](https://app.supabase.com) directory
-
-   ```sh
-   cd studio
-   ```
-
-1. Install npm/yarn dependencies:
-
-   npm
+[Supabase](https://supabase.com) uses [Turborepo](https://turborepo.org/docs) to manage and run this monorepo.
 
+1. Install the dependences in the root of the repo.
    ```sh
-   npm install
+   npm install # install dependencies
    ```
 
-   or with yarn
-
+2. You can then run the apps simultaneously with the following.
    ```sh
-   yarn install
+   npm run dev # start all the applications
    ```
 
-## Start a development server
-
-To debug code and to see your changes in real time, it is often useful to have a local HTTP server. Click one of the three links below to choose which development server you want to start.
+Then visit, and edit, any of the following sites:
 
-- [Supabase Website](#supabase-website)
-- [Supabase Docs](#supabase-docs)
-- [Supabase Studio](#supabase-studio)
+Site | Directory | Scope name | Description | Local development server
+---- | --------- | ---------- | ----------- | ------------------------
+[supabase.com](https://supabase.com) | `/apps/www` | www | The main website | http://localhost:3000
+[app.supabase.com](https://app.supabase.com) | `/studio` | studio | Studio dashboard | http://localhost:8082
+[POC] Next.js based Docs site | `/apps/temp-docs` | temp-docs | Guides and Reference (Next.js based) | http://localhost:3001  
+[POC] Community forum | `/apps/temp-community-forum` | forum | GitHub Discussions | http://localhost:3002
+[POC] DEV articles site | `/apps/temp-community-tutorials` | tutorials | DEV articles (which community members can write) | http://localhost:3003
```

Comment:
Should we hide these from Turbo for now? Makes it easier for contributors

Reply:
@saltcod @joshenlim @alaister just do another PR if this isn't right. 
going to just merge this for now.

Reply:
yep, we've hidden these now